<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>414</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Simple UI</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>5</number>
     </property>
     <item>
      <widget class="QLabel" name="example_lb">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600; font-style:italic;&quot;&gt;Title:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="example_le">
       <property name="placeholderText">
        <string>Type Something...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="example_te"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>5</number>
     </property>
     <item>
      <widget class="QPushButton" name="example_bttn">
       <property name="text">
        <string>Button</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="a_radio">
       <property name="text">
        <string>a</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">button_grp_1</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="b_radio">
       <property name="text">
        <string>b</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">button_grp_1</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="c_radio">
       <property name="text">
        <string>c</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">button_grp_1</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="d_radio">
       <property name="text">
        <string>d</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">button_grp_1</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="example_check">
       <property name="text">
        <string>Check</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QSlider" name="horizontalSlider">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="example_sb"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>horizontalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>example_sb</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>125</x>
     <y>276</y>
    </hint>
    <hint type="destinationlabel">
     <x>386</x>
     <y>281</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>example_sb</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>375</x>
     <y>272</y>
    </hint>
    <hint type="destinationlabel">
     <x>244</x>
     <y>278</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>setStackedID1()</slot>
 </slots>
 <buttongroups>
  <buttongroup name="button_grp_1"/>
 </buttongroups>
</ui>
